monitor/mqtt_monitor_test.go

```diff
@@ -0,0 +1,105 @@
+package monitor_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/planetmint/planetmint-go/config"
+	"github.com/planetmint/planetmint-go/monitor"
+	"github.com/planetmint/planetmint-go/util/mocks"
+	"github.com/stretchr/testify/assert"
+	"github.com/syndtr/goleveldb/leveldb"
+)
+
+func init() {
+	// Use MQTT mock client
+	monitor.MonitorMQTTClient = &mocks.MockMQTTClient{}
+}
+
+const (
+	challengerInput = "plmnt1fx3x6u8k5q8kjl7pamsuwjtut8nkks8dk92dek"
+	challengeeInput = "plmnt1fsaljz3xqf6vchkjxfzfrd30cdp3j4vqh298pr"
+)
+
+func TestGMonitorActiveParticipants(t *testing.T) {
+	monitor.LazyLoadMonitorMQTTClient()
+	cfg := config.GetConfig()
+	db, err := leveldb.OpenFile("./activeActors.db", nil)
```

Comment:
I would either replace this with:
`db, err := leveldb.Openfile(storage.NewMemoryStorage(), nil)`

or add the create Path to `.gitignore`